### PR TITLE
Adding first few Angular views

### PR DESCRIPTION
**Pull request by holtbp**
Sidekick: @rantler or @jstenhouse 

Goals:
- Add Angular
- Create first couple Angular views/routes
- Fix issues with relations in fixtures

### CODE REVIEW

File: app/assets/javascripts/templates/beers/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        %strong Style
+      %td
+        %strong Country
+  %tr{:'ng-repeat' => "beer in beers"}
+    %td.name
+      %a{:'ng-click' => "viewBeer(beer.id)"}
+        {{ beer.name }}
+    %td.brewer {{ beer.brewer.name }}
+    %td.style {{ beer.style.name }}
+    %td.country {{ beer.country.name }}
```
Comment (jstenhouse) [inline app/assets/javascripts/templates/beers/index.html.haml:20]: You want to remove the other haml view files in `/app/views/beers`? I'm guessing we won't need any of those right?

File: app/controllers/beers_controller.rb [hunk 2]
```diff
@@ -5,11 +5,20 @@ class BeersController < ApplicationController
...
+    respond_to do |format|
+      format.html { render 'index' }
+      format.json { render json: @beers.to_json }
+    end
   end
 
   # GET /beers/1
   # GET /beers/1.json
   def show
+    @beer = Beer.find(params[:id])
```
Comment (jstenhouse) [inline app/controllers/beers_controller.rb]: There's a `before_action` to set the beer for [:show, :edit, :update, :destroy], so the find isn't needed.

**Comment by jstenhouse** [comment]
Looks neato. I need to play with angular more now. (a couple minor comments) :ship: :it:

**Comment by holtbp** [comment]
Both good suggestions. I will take care of them before merging.

**Pull request merged**